tools/validators/instance_validator/setup.py

```diff
@@ -28,8 +28,10 @@
     'Nigel Kilmer',
     packages=find_packages(),
     install_requires=[
+        'protobuf==3.20.0rc2', 'proto-plus<1.19.7',
         'ruamel.yaml==0.17.4', 'strictyaml==1.4.2',
-        'google-cloud-pubsub', 'googleapis-common-protos',
-        'google-auth', 'protobuf', 'proto-plus'],
```

Comment:
keep those, I think thats why your tests are failing

Reply:
yup, reverted.  Also changed python version to >= 3.9